apps/src/gamelab/AnimationTab/ResizablePanes.jsx

```diff
@@ -13,14 +13,15 @@ var _ = require('../../lodash');
  */
 var ResizablePanes = React.createClass({
```

Comment:
I had to refactor `ResizablePanes` because my `AnimationPreview` component has to be given explicit dimensions, which can change now that it's in a resizable column.  This wasn't broadcasting any change on resize.  To fix that, it no longer stores column sizes as internal state, and instead acts like many React inputs, getting column sizes as a prop and pushing the new values to an onChange handler that is responsible for eventually updating the props.